### PR TITLE
fix(bash): return the newly installed version

### PR DESCRIPTION
**Pull request by clementprevot**
This way, it's possible to execute a `nvm use` with the
new installed version so that the parent shell can use
this new version right away.

Also, update the doc for the auto-switching.

### CODE REVIEW

**Comment by clementprevot** [comment]
@Kikobeats This is my solution for the bash problem.
Be careful, this PR also embed the `migrate` one. So merge it after merging `migrate`

File: README.md
Comment (Kikobeats) [inline README.md]: this documentation is very closest with the nvm problem. Write this in a separate user case

**Review by Kikobeats — CHANGES_REQUESTED** [review]

File: README.md
Comment (clementprevot) [reply README.md]: OK! I'll do that tomorrow!

Comment (clementprevot) [reply README.md]: Done

File: bin/index.js
Comment (Kikobeats) [inline bin/index.js:53]: why console.log?

Comment (clementprevot) [reply bin/index.js:53]: To have the version echoed to the stdout and then beeing able to get it in a bash variable

Comment (Kikobeats) [reply bin/index.js:53]: This maybe have sense for `nvm` but not for `n`. I'm going test it using `n`

Comment (clementprevot) [reply bin/index.js:53]: I made it in a way that it will not change anything except that there is a line echoed to the terminal with the version of the node used/installed, so it should not change anything for `n` except for this display

Comment (Kikobeats) [reply bin/index.js:53]: The point is, it's a very tricky solution for nvm that affects n users. 

I understand that you need the echo for setup correctly nvm, but, as n user, I don't need it.

One question, are you considered move from nvm to n?

As you can see, a lot of problems related with nvm.

Comment (clementprevot) [reply bin/index.js:53]: I understand. I'm not considering moving to n right now, but maybe in the futur.
You don't have to merge this pull request. It was just a suggestion. And it may help anyone that has this problem to solve it

Comment (Kikobeats) [reply bin/index.js:53]: I can't merge this code because is a particular problem resolved as generic :(

**Pull request closed without merging**
